packages/debs/arm64/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM arm64v8/ubuntu:jammy
```

Comment:
Warn: this OS will set a glibc baseline version of 2.35 (just guessing, no truly checked). Despite this is also used at AMD archs, so validate this with 5.0 compatibility matrix

Reply:
We decided to update the base OS for the RPM builder image because CentOS 7 doesn't have gcc11. For the DEB one, we will continue with Ubuntu 22.04.